clang/lib/CIR/Dialect/Transforms/TargetLowering/MissingFeature.h

```diff
@@ -0,0 +1,43 @@
+//===--- MissingFeatures.cpp - Markers for missing C/C++ features in CIR --===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Static methods declared here are spread throughout the CIR lowering codebase
+// to track missing features in CIR that should eventually be implemented. Add
+// new methods as needed.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+namespace mlir {
+namespace cir {
+
+struct MissingFeature {
```

Comment:
We already have something in `clang/lib/CIR/Dialect/IR/MissingFeatures.h`, please unify somehow.

Reply:
This is focused on missing features of the `TargetLowering` library, not the IR in general. Still, I can merge them if you prefer.

Reply:
Ok. I'll create a separate PR merging the two existing `MissingFeatures`, then update this PR.

Reply:
@bcardosolopes :point_right: https://github.com/llvm/clangir/pull/649

Reply:
@bcardosolopes :point_right: https://github.com/llvm/clangir/pull/649